docs/DeveloperGuide.md

```diff
@@ -173,9 +174,9 @@ The `Model` component handles the data and state management for the app, storing
     
 3. **Responsibilities**:
     * **Data Storage**:
-        * Maintains address book data, including lists of `Person`, `Wedding`, and `Task` objects.
+        * Maintains address book data, including lists of `Person`, `Wedding`, `Tag` and `Task` objects.
     * **Filtered Views**:
-        * Provides filtered lists of `Person`, `Wedding`, and `Task` objects (e.g., search results), exposing these as unmodifiable `ObservableList`s. This allows the UI to automatically update in response to changes.
+        * Provides filtered lists of `Person`, `Wedding`, `Tag` and `Task` objects (e.g., search results), exposing these as unmodifiable `ObservableList`s. This allows the UI to automatically update in response to changes.
```

Comment:
I'm not sure this is correct, we don't provide filtered views of wedding, tag, or task objects, we only filter for Person.